### DISCUSSION TITLE
Unable to change any parameters in debug mode list

### DISCUSSION THREAD

**Discussion by Lola-Ice** [lllyasviel/Fooocus#1569] [Q&A]
Any attempt to change debug mode list are met with the cursor becoming a white "forbidden" icon. This applies to all of the list such as guidance, sampler, scheduler etc. The debug mode is checked but I still am unable to change anything. Any ideas how to disable the disabler????

**Comment by mashb1t** [comment]
You're most likely using the "Extreme Speed" performance preset, where these options aren't available as they're set to a fixed value.
Choosing another Performance preset should do the trick.

**Reply by Lola-Ice** [reply]
Thank you very much. I am indeed using "Extreme speed". I will do as you say.
